lib/services/thinkingParser.js

```diff
@@ -34,17 +35,19 @@ var errors = require('../errors'),
  * @param {Object} result           Object representing the structured request.
  */
 
-function parseHumidity(fields, result, callback) {
+function parseHumidity(fields, result, timestamp, callback) {
     if (fields.length >= 5) {
         result.attributes.push({
             name: 'humidity',
             value: fields[3],
-            type: 'float'
+            type: 'float',
+            metadatas: timestamp
```

Comment:
Perfect! ;) BTW, I just saw a couple of missing final blank lines in a couple of files... :) I'll comment on the precise line... ;) Thanks!